### PR TITLE
Add Squash feature on GitFlow Plugin Form

### PR DESCRIPTION
**Pull request by Fabman08**
## Proposed changes
Add new checkbox on GitFlow plugin: "Squash feature"
This checkbox enables the user to squash the merge commit form one feature to develop branch.
Simply adds the -S parameter to the git flow feature finish command

I think it can be userful :)

## Screenshots 
image

## Test environment(s) 

- GIT 3.2.1.6628
- Windows 10

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Comment by pmiossec** [comment]
You have to update the translation, also... 

https://github.com/gitextensions/gitextensions/wiki/Translations

File: Plugins/GitFlow/GitFlowForm.Designer.cs [hunk 1]
```diff
@@ -444,10 +445,20 @@ private void InitializeComponent()
...
             this.lblRunCommand.TabIndex = 1;
             this.lblRunCommand.Text = "-";
             // 
+            // cbSquashFeature
+            // 
+            this.cbSquashFeature.AutoSize = true;
+            this.cbSquashFeature.Location = new System.Drawing.Point(492, 113);
+            this.cbSquashFeature.Name = "cbSquashFeature";
+            this.cbSquashFeature.Size = new System.Drawing.Size(98, 17);
+            this.cbSquashFeature.TabIndex = 9;
```
Comment (RussKie) [inline Plugins/GitFlow/GitFlowForm.Designer.cs]: Can you please confirm the tab order is logical.
If not, please fix it.

Comment (Fabman08) [reply Plugins/GitFlow/GitFlowForm.Designer.cs]: Hi @RussKie, I confirm the number 9 as TabIndex for `cbSquashFeature`

**Comment by Fabman08** [comment]
> You have to update the translation, also...
> 
> https://github.com/gitextensions/gitextensions/wiki/Translations

@pmiossec I've tried to launch the Power Shell command but I've no more files to commit and I've encountered the following error
image

image

Any idea?
Thank you!

**Comment by pmiossec** [comment]
Like that, no, I don't have an idea.

Please be sure to have made a full rebuild of the solution before running
the script.

**Comment by RussKie** [comment]
You also need to provide your sign off in contributors.txt

**Comment by pmiossec** [comment]
@Fabman08 I am trying to reproduce the crash in tha translation app without success.

In the mean time, you could find the updated translation commit in the branch: https://github.com/pmiossec/gitextensions/tree/Fabman08/master

**Comment by RussKie** [comment]
Just noticed, please abstain from working off the master branch or any other branches that you haven't created. 
Consider creating your own branches; otherwise you will find yourself in a world of pain when remote branches move and you pull down changes.

**Comment by RussKie** [comment]
Please rebase and squash, and it is good to go

**Comment by Fabman08** [comment]
@RussKie @pmiossec sorry, do I have to merge `contributors.txt` again?

**Comment by pmiossec** [comment]
Thank you!

**Pull request merged**